core/templates/components/state-editor/state-editor-properties-services/state-property.service.ts

```diff
@@ -70,6 +72,7 @@ export class StatePropertyService<StatePropertyType> {
     // 'saved' means that this is the latest value of the property as
     // determined by the frontend change list.
     this.savedMemento = cloneDeep(value);
+    this.statePropertyInitializedEmitter.emit();
```

Comment:
Have you updated the frontend tests to test this?

Reply:
Using an emitter is an implemetation detail, since we focus on testing interfaces, I didn't think it was necessary to test this. I've now added an expect statement for this in the relevant test.